base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -29,11 +29,11 @@ const (
 
 var (
 	ErrFailedPickVGF = gfsperrors.Register(VirtualGroupManagerSpace, http.StatusInternalServerError, 540001,
-		"failed to pick virtual group family, need creating global virtual group")
+		"failed to pick virtual group family, need create global virtual group")
```

Comment:
fix `need create` as `fix need to create`

Reply:
Fixed.